scripts/analysis.py

```diff
@@ -250,229 +116,198 @@ def get_sim_time_duration(cursor):
     timestep: list
         timeseries up to duration
     """
-    cur = cursor
-
-    info = cur.execute('SELECT initialyear, initialmonth,'
-                       + ' duration FROM info').fetchone()
+    info = cursor.execute('SELECT initialyear, initialmonth, '
+                          'duration FROM info').fetchone()
     init_year = info[0]
     init_month = info[1]
     duration = info[2]
-    timestep = np.linspace(0, info[2]-1, num=info[2])
+    timestep = np.linspace(0, info[2] - 1, num=info[2])
 
     return init_year, init_month, duration, timestep
 
 
-def isotope_mass_time_list(resources, compositions):
-    """Creates an list with isotope name, mass, and time
-
-    Parameters
-    ----------
-    resources: list
-        resource data from the resources table
-    compositions: list
-        composition data from the compositions table
-
-    Returns
-    -------
-    list
-        isotope name list
-    list
-        isotope mass list
-    list
-        isotope transaction time list
-
-    """
-
-    temp_isotope = []
-    temp_mass = []
-    time_list = []
-    for res in resources:
-        res_qualid = res[2]
-        indices = [x for x, y in enumerate(compositions) if y[0] == res_qualid]
-        for index in indices:
-            nucid = compositions[index][1]
-            mass_frac = compositions[index][2]
-            mass_waste = res[0]
-            res_time = res[1]
-            temp_isotope.append(nucid)
-            temp_mass.append(mass_frac*mass_waste)
-            time_list.append(res_time)
-
-    return temp_isotope, temp_mass, time_list
-
-
-def plot_in_out_flux(cursor, facility, influx_bool, title, outputname):
-    """plots timeseries outflux from facility name in kg.
+def get_timeseries(in_list, duration, multiplyby, cumulative):
+    """ returns a timeseries list of a given data
```

Comment:
Sorry, 'data' there was supposed to refer to in_list. I must have missed that.
I''ll fix this as well.